src/installer.js

```diff
@@ -133,6 +133,21 @@ var getTrashDepends = function (options, callback) {
 }
 
 /**
+ * Determine the default dependencies for an Electron application.
+ */
+var getDepends = function (trashDependencies) {
+  return [
+    trashDependencies,
+    'libgconf2-4',
+    'libgtk2.0-0',
+    'libnotify4',
+    'libnss3',
+    'libxtst6',
+    'xdg-utils'
+  ]
```

Comment:
So basically, the only reason this function exists it's too be able to link it on the README? So the user doesn't have to scroll up and find out what trashDependencies are. Otherwise, this function is pretty useless. Is this correct?

Reply:
Currently, yes. Pretty much aesthetic.